### ISSUE TITLE
`antidote load` does not load setopts from plugins

### ISSUE THREAD

**Issue by erfanara** [mattmc3/antidote#36]
# Problematic configuration
- simple `.zshrc` file:
```
if ! [[ -e ${ZDOTDIR:-~}/.antidote ]]; then
  git clone https://github.com/mattmc3/antidote.git ${ZDOTDIR:-~}/.antidote
fi
source ${ZDOTDIR:-~}/.antidote/antidote.zsh
antidote load
```

-  simple`.zsh_plugins.txt` file:
```
zsh-users/zsh-completions
mattmc3/zephyr
```

- now `setopt` output:
```
interactive
monitor
shinstdin
zle
```

- adding `source .zsh_plugins.zsh` into the `.zshrc` fixes the issue
- somehow `antidote load` can't source `.zsh_plugins.zsh` fully

**Comment by mattmc3** [comment]
Looks like this is a regression from this fix: https://github.com/mattmc3/antidote/issues/28

I'll have a look.

**Comment by mattmc3** [comment]
I added a test for #28 and then removed the zsh emulation from the load command. If you grab the latest (run `antidote update`), it should be resolved for you. Let me know if it isn't.

**Comment by erfanara** [comment]
I tested again with the latest, but It is not fixed yet.

**Comment by erfanara** [comment]
I removed zsh emulation and local_options from `_antidote_main` and `antidote` too, now it works.

**Comment by mattmc3** [comment]
Thanks! Great feedback. This ones a pesky bug as I don't want to introduce a fix here that breaks #28 again. Let me do a bit more testing. I hope to push a fix today.

**Comment by mattmc3** [comment]
@erfanara - pretty sure I've got it covered this time 🤞 . Please upgrade to v1.4
